app/templates/gulp/_env-config.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var gulp = require('gulp');
+var fs = require('fs');
+var argv = require('yargs').argv;
+var $ = require('gulp-load-plugins')();
+
+//var ngConstant = require('gulp-ng-constant');
+
+var environment = argv.env || 'default';
+//var ENV = JSON.parse(fs.readFileSync('./src/app/env_configs/config-' + environment + '.json', 'utf8')).ENV;
+
+gulp.task('env-config', function () {
+ return gulp.src('src/app/env_configs/config-' + environment + '.json')
```

Comment:
This line should read from the config

    return gulp.src(path.join(conf.paths.src, '/app/env_configs/config-' + environment + '.json'))